views/blocks/wirecard_credit_card_fields.tpl

```diff
@@ -31,14 +32,24 @@
       <input type="hidden" name="challenge" value="[{$challenge}]">
       <input type="hidden" name="sDeliveryAddressMD5" value="[{$oView->getDeliveryAddressMD5()}]">
       [{if $oView->isActive('PsLogin') || !$oView->isConfirmAGBActive()}]
-    <input type="hidden" name="ord_agb" value="1">
+        <input type="hidden" name="ord_agb" value="1">
```

Comment:
is it not possible to put the if statement as a print to the value? then we have only one input in the code?

Reply:
This code has not been changed (only its indentation), which is why we would consider it as an out-of-scope refactoring that should not be part of this PR.